addons/mail/static/src/components/thread_view_topbar/thread_view_topbar.scss

```diff
@@ -23,6 +23,14 @@
     background-color: $gray-300;
 }
 
+.o_ThreadViewTopbar_button_Active {
+    background-color: $gray-300;
+}
+
+.o_ThreadViewTopbar_button_Active:hover {
+    background-color: $gray-400;
+}
```

Comment:
I prefer putting modifier classname `o-isActive`, and have SCSS show as modifiers:
```scss
.o_ThreadViewTopbar_button {
   &:not(.o-isActive):hover { bg-c: $gray-300; }
   &.o-isActive { bg-c: $gray-300 ; }
   &.o-isActive:hover { bg-c: $gray-400; }
}
```